config/subsidy-application-management/forms/urban-renewal/application-phase2/versions/20221218090000/form.ttl

```diff
@@ -485,9 +490,464 @@ ext:ProjectfaseringMetProjectverloopTimingEnRisicobeheerProjectGroup
+          form:scope ext:projectOnderdeelStadListingS;
+          form:createGenerator ext:projectOnderdeelStadListingGenerator;
+          sh:group ext:projectOnderdeelAnalysePg;
+          form:canAdd true;
+          form:addLabel "Voeg stap toe";
+          form:canRemove true;
+          form:removeLabel "Verwijder";
+          sh:maxCount 1;
+          sh:order 2 .
+
+        ##########################################################
+        #  Subform: projectOnderdeelStadListingUnit
+        ##########################################################
+        ext:projectOnderdeelStadListingSf a form:SubForm;
+          form:includes ext:projectOnderdeelStadListingUnitAandeelF;
+          form:includes ext:projectOnderdeelStadListingUnitBedragF;
+          form:hasFieldGroup ext:projectOnderdeelAnalysePg.
+
+
+        ##########################################################
+        # Field: Projectonderdeel Stad Listing Aandeel Input
+        ##########################################################
+        ext:projectOnderdeelStadListingUnitAandeelF
+            a                 form:Field ;
+            sh:name           "Eigen aandeel" ;
+            sh:order          1 ;
+            sh:path           lblodSubsidie:projectOnderdeelStadListingUnitAandeel ;
+            form:validations
+                              [ a                 form:MaxLength ;
+                                form:grouping     form:MatchEvery ;
+                                form:max          "500" ;
+                                sh:resultMessage  "Max. karakters overschreden." ;
+                                sh:path            lblodSubsidie:projectOnderdeelStadListingUnitAandeel
+                              ];
+            form:displayType  displayTypes:defaultInput ;
+            sh:group          ext:projectOnderdeelAnalysePg .
+
+        ##########################################################
+        # Field: Projectonderdeel Stad Listing Bedrag Input
+        ##########################################################
+        ext:projectOnderdeelStadListingUnitBedragF
+            a                 form:Field ;
+            sh:name           "Bedrag" ;
+            sh:order          2 ;
+            sh:path           lblodSubsidie:projectOnderdeelStadListingUnitBedrag ;
+            form:displayType  displayTypes:numericalInput ;
+            sh:group          ext:projectOnderdeelAnalysePg .
+
+        ##########################################################
+        #  Generator: projectOnderdeel Stad Listing
+        ##########################################################
+        ext:projectOnderdeelStadListingGenerator a form:Generator;
+          form:prototype [
+            form:shape [
+              a lblodSubsidie:projectOnderdeelStadUnit ;
+              lblodSubsidie:projectOnderdeelStadListingUnitAandeel "" ;
+              lblodSubsidie:projectOnderdeelStadListingUnitBedrag "" ;
```

Comment:
I would either set this to 0 or don't set it since it's a numerical input?

```suggestion
              lblodSubsidie:projectOnderdeelStadListingUnitBedrag 0 ;
```